src/openepd/model/specs/generated/conveying_equipment.py

```diff
@@ -31,6 +31,23 @@ class EscalatorsV1(BaseOpenEpdHierarchicalSpec):
     which conveys people between the floors of a building.
     """
 
+    vertical_rize: LengthMStr | None = pyd.Field(
+        default=None,
+        description="The vertical distance between the top and bottom landings of an escalator",
+        example="1 m",
+    )
+    speed: SpeedStr | None = pyd.Field(default=None, description="Reference speed of the escalator", example="1 m / s")
+    step_width: LengthMStr | None = pyd.Field(default=None, description="Width of the escalator steps", example="1 m")
+    max_capacity: CapacityPerHourStr | None = pyd.Field(
+        default=None, description="Max capacity of escalator in persons per hour", example="1 hour^-1"
```

Comment:
It's discussable. I really don't expect people to recalculate such properties as people per second. I feel like we are duplicating time units, but there is great difference between seconds, hours and years

Reply:
@alexlitvinenko 

I think we agreed on the fact that material spec extensions will be much much more relaxed than the main openEPD - otherwise it stops having sense. For example, we can technically measure rebar diameter or concrete slump in meters, and would have `0.008 m` but it is very inconvenient for the consumers and everyone would have to convert.